src/ui/.dockerignore

```diff
@@ -1,6 +1,10 @@
 .env
+.env.sample
+.gitignore
 .prettierrc.json
+.prettierignore
+.eslintrc.json
 .eslintignore
-node_modules
```

Comment:
we should keep the node_modules in the `.dockerignore` file as well to avoid copying it to the container

Reply:
Good shout have added.